Data/crypto/readme.md

```diff
@@ -0,0 +1,127 @@
+![alt tag](https://raw.githubusercontent.com/QuantConnect/Lean/master/Documentation/logo.white.small.png) 
+## LEAN Data Formats / Cryptocurrency (crypto)
+
+QuantConnect hosts crypto data provided by [Kaiko](https://www.kaiko.com).
+The data contains both *Trade* and *Quote* data. Using the ToolBox applications `GDAXDownloader` and `BitfinexDownloader`, you can obtain historical *trade* data for free, but not *quote* data with this method. 
+You can also download crypto data (trades and quotes starting with Tick and ending with Minute resolution) for a fee on our website. You can explore the data and purchase it at https://www.quantconnect.com/data/tree/crypto
+
+CSV files are stored in compressed zip files, each containing a single CSV file.
+
+Crypto data supports the following Resolutions:
+
+* Tick
+* Second
+* Minute
+* Hour
+* Daily
+
+The markets we currently support are: 
+
+* GDAX/Coinbase Pro - 24/7 Market
+* Bitfinex (Beta) - 24/7 Market
+
+For crypto data, `tickType` can be:
+
+* trade
+* quote
+
+All times are in UTC unless noted otherwise.
+
+## Tick Format
```

Comment:
Please apply same rules as the CFD comments to all the remaining files -- "Minute and Second File Format" first; these are the most popular resolutions so we make them easiest to find. The "Hour and Daily File Format" then "Tick File Format"